src/main/java/step2/strategy/RandomDrawStrategy.java

```diff
@@ -0,0 +1,17 @@
+package step2.strategy;
+
+import java.util.Random;
+
+public class RandomDrawStrategy implements DrawStrategy{
+    private static final Random rand = new Random();
+    public static final int MAXIMUM_BOUND = 100;
+    public static final int RANDOM_CONDITION = 50;
+
+    @Override
+    public boolean draw() {
+        if(rand.nextInt(MAXIMUM_BOUND) > RANDOM_CONDITION) {
+            return true;
+        }
+        return false;
```

Comment:
```suggestion
        return rand.nextInt(MAXIMUM_BOUND) > RANDOM_CONDITION);
```
boolean은 바로 반환해주면 된답니다!